app/components/attachments/table_component.html.erb

```diff
@@ -0,0 +1,209 @@
+          text-xs uppercase text-slate-700 bg-slate-50 dark:bg-slate-700
+          dark:text-slate-300 sticky top-0 z-10
+        '
+      >
+        <tr>
+          <% @columns.each_with_index do |column, index| %>
+            <%= render_cell(
+              tag: 'th',
+              scope: 'col',
+              classes: class_names('px-3 py-3', 'sticky left-0 min-w-56 max-w-56 z-10 bg-slate-50 dark:bg-slate-700': index.zero?, 'sticky left-56 z-10 bg-slate-50 dark:bg-slate-700': index == 1)
+            ) do %>
+              <% if index.zero? and @abilities[:select_attachments] %>
+                <%= check_box_tag "select-page",
+                title: t(:".select_page"),
+                "aria-label": t(:".select_page"),
+                data: {
+                  action: "input->selection#togglePage",
+                  controller: "filters",
+                  selection_target: "selectPage",
+                },
+                class:
+                  "w-4 h-4 mr-2.5 text-primary-600 bg-slate-100 border-slate-300 rounded focus:ring-primary-500 dark:focus:ring-primary-600 dark:ring-offset-slate-800 focus:ring-2 dark:bg-slate-700 dark:border-slate-600" %>
+              <% end %>
+              <% if column == :id %>
+                <%= render Ransack::SortComponent.new(
+                  ransack_obj: @q,
+                  label: t(".#{column}"),
+                  url: helpers.sorting_url(@q, "puid"),
+                  field: "puid",
+                  data: {
+                    turbo_action: "replace",
+                  },
+                ) %>
+              <% elsif column == :byte_size || column == :filename %>
+                <%= render Ransack::SortComponent.new(
+                  ransack_obj: @q,
+                  label: t(".#{column}"),
+                  url: helpers.sorting_url(@q, "file_blob_#{column}"),
+                  field: "file_blob_#{column}",
+                  data: {
+                    turbo_action: "replace",
+                  },
+                ) %>
+              <% elsif column == :format || column == :type %>
+                <%= render Ransack::SortComponent.new(
+                  ransack_obj: @q,
+                  label: column,
+                  url:
+                    helpers.sorting_url(@q, URI.encode_www_form_component("metadata_#{column}")),
+                  field: "metadata_#{column}",
+                  data: {
+                    turbo_action: "replace",
+                  },
+                ) %>
+              <% else %>
+                <%= render Ransack::SortComponent.new(
+                  ransack_obj: @q,
+                  label: t(".#{column}"),
+                  url: helpers.sorting_url(@q, column),
+                  field: column,
+                  data: {
+                    turbo_action: "replace",
+                  },
+                ) %>
+              <% end %>
+            <% end %>
+          <% end %>
+          <% if @renders_row_actions %>
```

Comment:
**nitpick:** Can you hide this column until the functionality is implemented?  We don't want non-functional code being displayed to a user.

Reply:
Commented out the call to render the destroy action where the table is called in eb840b82dfab6d6086f83d6e497b6767237f96c3

Adding destroy functionality will be my next PR